### ISSUE TITLE
 Acme agents hyperparameters

### ISSUE THREAD

**Issue by rasoolfa** [deepmind/acme#53]
Hi,

Do the Acme agents in "acme/examples/" follow the same setups and hyper-parameters as their corresponding papers? can those exact same codes be used to reproduce their results? or need some changes like reward scaling, hyper-parameters, etc?

Thanks.

**Comment by fastturtle** [comment]
Hi @rasoolfa while the hyperparameters in these agents may be similar to their corresponding papers, they are not identical. I would recommend looking at the hyperparameters used in the papers before attempting to exactly reproduce any results.

**Comment by rasoolfa** [comment]
Thanks @fastturtle. 
On the same note, would it be possible to share more info about hyper-parameters method like D4PG? For example, D4PG doesn't include information about all of important hyperparameters in the paper such as **vmin** and **vmax** . Although there is a [note](https://github.com/deepmind/acme/tree/799f76fffe8dde4e6bbf3c6cb1c0260cf7ff00aa/acme/agents/tf/d4pg) says how to find them, it can still lead to very different results and it is kinda unclear. 
If it is not possible to share them, can you clarify a bit the note about vmin and vmax? 
"A good rule of thumb is to set vmax to the discounted sum of the maximum instantaneous rewards for the maximum episode length;" What policy should be used to gather this info and when? random policy? or we should train an agent for a while with another method and then use that policy it to get maximum reward? how this vmax/vmin related to the final an agent's returns and optimal policy? For example, final return of Acrobot(Swingup) is ~400 (Figure 2 in the paper), so should vmax be very close to 400?
